meinberlin/apps/budgeting/assets/FilterBar.jsx

```diff
@@ -71,24 +71,22 @@ export const FilterBar = props => {
         <FilterBarListMapSwitch query={filterString} />
       </div>
       <div className="offset-lg-2 col-lg-8">
-        <div className="filter-bar__section">
-          <div className="filter-bar__section-item">
+        <div className="control-bar control-bar-search">
+          <div className="searchinput">
             <FilterBarSearch
               term={term}
               onSearch={value => handleSearch(value)}
             />
           </div>
           {props.filters?.ordering && (
-            <div className="filter-bar__section-item">
-              <FilterBarDropdown
-                key="ordering_dropdown"
-                filter={props.filters.ordering}
-                filterId="id_ordering"
-                onSelectFilter={choice => applyFilter('ordering', choice)}
-              />
-            </div>
+            <FilterBarDropdown
+              key="ordering_dropdown"
+              filter={props.filters.ordering}
+              filterId="id_ordering"
+              onSelectFilter={choice => applyFilter('ordering', choice)}
+            />
           )}
-          <div className="filter-bar__section-item">
+          <div className="togglebutton">
```

Comment:
sorry @phillimorland, but not sure if i get it all right.

yes i wasn't sure about these class names, i thought i use something that fits to the existing one `dropdown`.
I know `dropdown` is kind of a keyword in BS5 and is used as a `modifier`. So bs5 is not BEM right?

the mess (at least in my head) comes with mixing bootstrap 5 and BEM. So bs5 uses `dropdown` to identify functionality/modifier. But here it gets overwritten and styled (margin-bottom).

Additionally we seem to not following BEM regarding spacing. Because buttons, dropdowns and general elements have spacing applied. As i understand BEM, spacing should be the responsibility of the container?

>Also to match out existing convention we should either use filter-bar as out block element and extend to control bar in the component style file or change the name of the component.

Not sure if I follow this correctly. So i am creating a new class `filter-bar` and use `control-bar` as base and would have something like `filter-bar__search` and `filter-bar__btn-toggle`?